src/main/java/com/SAS/User/User.java

```diff
@@ -4,10 +4,12 @@
 
 package com.SAS.User;
 
+import java.util.HashSet;
+
 public abstract class User {
 
     public static int counter = 0;
-
+    private HashSet<String> myPrivileges;
     private int userID;
```

Comment:
the userID is to identify guest users or registered users? or both?

Reply:
We'll remove the userID after we'll have the DB.